swift/Arrow/Sources/Arrow/ArrowReader.swift

```diff
@@ -216,6 +216,75 @@ public class ArrowReader { // swiftlint:disable:this type_body_length
         return .success(RecordBatch(arrowSchema, columns: columns))
     }
 
+    /*
+     The Memory stream format is for reading the arrow streaming protocol.  This
+     format is slightly different from the File format protocol as it doesn't contain
+     a header and footer
+     */
+    public func fromMemoryStream( // swiftlint:disable:this function_body_length
```

Comment:
How about using `readStreaming` (for the Arrow streaming format) and `readFile` (for the Arrow file format) instead of `fromMemoryStream` (for the Arrow streaming format) and `fromStream` (for the Arrow file format)?

```suggestion
    /*
     This is for reading the Arrow streaming format. The Arrow streaming format
     is slightly different from the Arrow File format as it doesn't contain a header
     and footer.
     */
    public func readStreaming( // swiftlint:disable:this function_body_length
```